scripts/common.py

```diff
@@ -112,7 +112,7 @@
 TEST_BRANCH_REGEX = r'test-[A-Za-z0-9-]*$'
 USER_PREFERENCES = {'open_new_tab_in_browser': None}
 
-FECONF_PATH = os.path.join('.', 'feconf.py')
+FECONF_PATH = os.path.join('feconf.py')
```

Comment:
Maybe do it for REDIS_CONF_PATH below too, for consistency.

Reply:
Done.